apollo-converters/pojo/src/main/java/com/apollographql/android/converter/pojo/ApolloConverterFactory.java

```diff
@@ -21,11 +27,26 @@
 public final class ApolloConverterFactory extends Converter.Factory {
   private final Map<Type, ResponseFieldMapper> responseFieldMappers;
   private final Map<ScalarType, CustomTypeAdapter> customTypeAdapters;
+  private final Moshi moshi;
```

Comment:
I'm not saying to generate parsing. I'd like to be open to having a ConverterFacotory which can be a gson or moshi factory.  Apollo client should be agnostic to how the parsing is done but instead just delegate to a user defined parser factory.